src/main/java/appeng/client/render/items/ToolColorApplicatorRender.java

```diff
@@ -156,7 +157,16 @@ else if( type == ItemRenderType.EQUIPPED )
 			tessellator.draw();
 		}
 
-		GL11.glPopAttrib();
+		GL11.glEnable( GL11.GL_LIGHTING );
+		// GL11.glPopAttrib();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)